src/Bookings.js

```diff
@@ -1,18 +1,47 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+import FakeBookings from "./data/fakeBookings.json";
+import CustomerProfile from "./CustomerProfile.js";
 
 const Bookings = () => {
+  const [bookings, setBookings] = useState([]);
+  const [profile, setProfile] = useState({});
+
+  useEffect(() => {
+    fetch("https://cyf-react.glitch.me/delayed")
+      .then(res => res.json())
+      .then(data => {
+        setBookings(data);
+      });
+  }, []);
+
+  function showProfile(profile) {
+    setProfile(profile);
+  }
+
   const search = searchVal => {
+    const filteredName = bookings.filter(
+      booking =>
+        booking.firstName.toLowerCase() === searchVal ||
+        booking.surname.toLowerCase() === searchVal
+    );
+    setBookings(filteredName);
     console.info("TO DO!", searchVal);
```

Comment:
It's best to avoid committing any console statements. 

They are sometimes ok for debugging as long as you only use them locally and remove them before pushing your changes.

If you don't keep on top of use of `console`, as the codebase grows you accumulate more and more of them and it can get messy and confusing, and annoying for other developers working on the codebase.

Reply:
Reminder for next checkin- I can show you how to use the Chrome debug tools so you don't need to do this.